### PR TITLE
 [Tests] Add a simple test for atlasprint

### PR DESCRIPTION
**Pull request by rldhont**
* Update the **test_print** project
* Ada a simple atlasprint test

Funded by 3Liz

### CODE REVIEW

File: tests/Makefile [hunk 1]
```diff
@@ -41,16 +41,18 @@ env:
 	EOF
 
 
-build-plugins: 
+build-plugins:
 	./add_server_plugins.sh
 
+load-plugins:
+	./load_server_plugins.sh
```
[the remark this answers is not in the document]
Comment (rldhont) [reply tests/Makefile:48]: The first one loading the zip from github. The second uses qgis-plugin-mnanager.

[the remark this answers is not in the document]
Comment (rldhont) [reply tests/Makefile:48]: And `add_server_plugin` install the master version of `Lizmap Server`

[the remark this answers is not in the document]
Comment (rldhont) [reply tests/Makefile:48]: #3217 is probably the right solution. I just need a project with `atlasprint` available for tests.

[the remark this answers is not in the document]
Comment (Gustry) [reply tests/Makefile:48]: I was talking about "english" point of view.
Both script are "adding" and "loading" QGIS server plugins (whatever their source github, plugins.qgis.org or packages.3liz.org) . So it doesn't make sense to have two scripts doing the same thing ;-)

I have updated #3217 (until I finish my work about plugins)

**Comment by Gustry** [comment]
Please rebase your PR on top of master and skip installation of plugins (GH action). It has been done in #3217

**Comment by rldhont** [comment]
@Gustry PR rebased but an error with the script `add_server_plugins.sh` in GH
```
Archive:  /tmp/lizmap_server.master.zip
checkdir:  cannot create extraction directory: /srv/plugins
           Permission denied
QGIS Server WfsOutputExtension and AlasPrint plugins
./add_server_plugins.sh: line 20: qgis-plugin-manager: command not found
Stable releases from http://plugins.qgis.org/
./add_server_plugins.sh: line 21: qgis-plugin-manager: command not found
./add_server_plugins.sh: line 23: qgis-plugin-manager: command not found
./add_server_plugins.sh: line 24: qgis-plugin-manager: command not found
Error: Process completed with exit code 127.
```

File: tests/end2end/cypress/integration/print-ghaction.js [hunk 2]
```diff
@@ -210,4 +210,12 @@ describe('Print', function () {
     //     })
     // })
 
+
+    it('should get atlas print', function () {
+        cy.get('#layer-quartiers button.btn.checkbox').click()
+        // Click quartier
+        cy.get('#map').click(480,340)
```
Comment (Gustry) [inline tests/end2end/cypress/integration/print-ghaction.js:217]:
```suggestion
        cy.mapClick(480,340)
```
Let's switch to this helper, make it easier one day if we change coordinates.

File: tests/qgis-projects/tests/test_print.qgs.cfg [hunk 3]
```diff
@@ -1,20 +1,22 @@
 {
     "metadata": {
-        "qgis_desktop_version": 31611,
-        "lizmap_plugin_version": "master",
-        "lizmap_web_client_target_version": 30400,
+        "qgis_desktop_version": 31616,
+        "lizmap_plugin_version_str": "3.8.3",
```
Comment (Gustry) [inline tests/qgis-projects/tests/test_print.qgs.cfg:4]: Even Lizmap developers are not upgrading their QGIS desktop lizmap plugin :)

File: tests/end2end/cypress/integration/print-ghaction.js (hunk 2 above)
Comment (rldhont) [reply tests/end2end/cypress/integration/print-ghaction.js:217]: I have done a copy/paste, I'll fix the other tests!

File: tests/qgis-projects/tests/test_print.qgs.cfg (hunk 3 above)
Comment (rldhont) [reply tests/qgis-projects/tests/test_print.qgs.cfg:4]: I don't use it often enough anymore

**Comment by rldhont** [comment]
@Gustry did you rebase my branch ?

**Comment by Gustry** [comment]
@rldhont I guess you would be very lucky if you could only "search and replace" like you did your latest commit.
It's not a straight search and replace, look at the code of `mapClick` : https://github.com/3liz/lizmap-web-client/blob/master/tests/end2end/cypress/support/commands.js#L60..L68
I would only change the one which was in your test.

BTW, your cypress seems hanging now ... https://github.com/3liz/lizmap-web-client/actions/runs/3103482121/jobs/5026862623
42 minutes running ...

**Comment by rldhont** [comment]
I have split the PR and created #3238 

@Gustry why did you remove backport_3_5 ?

**Comment by Gustry** [comment]
> @Gustry why did you remove backport_3_5 ?

Because for now, there isn't AtlasPrint on the 3.5 stack, because qgis-plugin-manager is not available on QGIS 3.16 docker image. Only starting from 3.22.

I can add the ZIP download with wget.

**Review by Gustry — APPROVED** [review]

**Pull request merged**